### PR TITLE
Update Media Browser Bar to show image on mobile but a bit less

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

previous:
image

New:
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by balloob** [comment]
Especially with more control buttons, I think that we should not show the image but as much name as possible.

**Comment by balloob** [comment]
I wonder if we should remove volume here. It is shown when people click the name/icon. And if we make the bar less high, the picture will also be less wide, allowing for more space for the title. Currently we can 1.5 words and that's too little

File: src/panels/media-browser/ha-bar-media-player.ts [hunk 1]
```diff
@@ -640,13 +646,14 @@ export class BarMediaPlayer extends LitElement {
       }
 
       :host([narrow]) img {
-        max-height: 80px;
-        max-width: 80px;
+        max-height: 56px;
+        max-width: 56px;
+        padding-left: 2px;
```
Comment (balloob) [inline src/panels/media-browser/ha-bar-media-player.ts]: images shouldn't have padding but margin. But shouldn't we do 0px ?

Comment (zsarnett) [reply src/panels/media-browser/ha-bar-media-player.ts]: I thought I removed this

Comment (zsarnett) [reply src/panels/media-browser/ha-bar-media-player.ts]: I think you are looking at old code

**Review by balloob — APPROVED** [review]

**Pull request merged**